client/spec/blog_settings_spec.js

```diff
@@ -60,5 +60,24 @@ describe('Blog settings', function() {
         openSettings();
         expectSelectedLanguageIs(DEFAULT_LANGUAGE);
     });
+    it('shows original creator full name', function() {
+        openBlog(0);
+        openSettings();
+        element(by.css('[data="blog-settings-users"]')).click();
+        //expect original creator name
+        element(by.css('[data="original-creator-display-name"]')).getText().then(function(text) {
+            expect(text).toEqual('first name last name');
+        });
+    });
+
+    it('shows original creator username', function() {
```

Comment:
I thought of that as well but I separated them for ease of visibility when one of the expectations are not met